client/src/main/java/io/getstream/chat/android/client/models/Attachment.kt

```diff
@@ -30,6 +32,14 @@ data class Attachment(
     var name: String? = null,
     var fallback: String? = null,
 
+    @IgnoreSerialisation
+    @IgnoreDeserialisation
+    var upload: File? = null,
```

Comment:
Adding this new field to LLC is a bit confusing.
Imagine someone using only LLC and they want to send an attachment.
They could do something like:
```
ChatClient.instance().sendMessage(
            channelType,
            channelId,
            Message(
                text = "Hello",
                attachments = mutableListOf(
                    Attachment(
                        upload = File(imagePath)
                    )
                )
            )
        )
```

Syntax is right, and the message will be sent without error. but no attachment will be upload to the server.